### PR TITLE
XW-1741 | update keys

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-1741

@Wikia/x-wing

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
https://github.com/Wikia/mercury/pull/2688

**Comment by kvas-damian** [comment]
LGTM

**Comment by Grunny** [comment]
LGTM.

**Comment by Warkot** [comment]
Remember to update these keys in Crowdin. If there are no untranslated strings it should be easy to upload them (including translations) via the cli tool. If not, translation memory can be used in Crowdin.

File: i18n/en/design-system.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
   "global-footer-fandom-follow-us-header": "Follow Us",
   "global-footer-fandom-link-fan-communities": "Fan Communities",
-  "global-footer-fandom-link-vertical-games": "Games",
-  "global-footer-fandom-link-vertical-movies": "Movies",
-  "global-footer-fandom-link-vertical-tv": "TV",
-  "global-footer-fandom-link-vertical-books": "Books",
-  "global-footer-fandom-link-vertical-comics": "Comics",
-  "global-footer-fandom-link-vertical-lifestyle": "Lifestyle",
-  "global-footer-fandom-link-vertical-music": "Music",
+  "global-footer-vertical-games": "Games",
```
Comment (rogatty) [inline i18n/en/design-system.json]: I don't think this is a good idea. Right until now we had 1:1 mapping API fields -> message keys.
Also, links in Fandom section are in different context from those in the bottom bar.
I'd revert this one and name the new one `global-footer-licensing-and-vertical-description-param-vertical-games`

**Comment by tomasznapieralski** [comment]
Crowdin updated in that point

**Comment by Warkot** [comment]
I see the changes are in Crowdin already. LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**